geode-core/src/main/java/org/apache/geode/internal/io/CoreSanctionedSerializablesService.java

```diff
@@ -12,20 +12,16 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package org.apache.geode.codeAnalysis;
+package org.apache.geode.internal.io;
 
+import java.net.URL;
 
-import org.apache.geode.internal.memcached.GeodeMemcachedService;
+import org.apache.geode.internal.serialization.SanctionedSerializablesService;
 
-public class AnalyzeMemcachedSerializablesJUnitTest extends AnalyzeSerializablesJUnitTestBase {
+public class CoreSanctionedSerializablesService implements SanctionedSerializablesService {
```

Comment:
Note: github diff is confused and the code is correct.

AnalyzeMemcachedSerializablesJUnitTest was renamed to AnalyzeMemcachedSerializablesIntegrationTest and still exists in geode-memcached.

CoreSanctionedSerializablesService is a new class which is completely unrelated.

Reply:
Fixed